package.json

```diff
@@ -1,43 +1,28 @@
 {
   "name": "create",
-  "version": "1.0.0",
+  "version": "3.0.0",
```

Comment:
Why not 2.0.0?

Reply:
because the release is currently at 2.0.1, and since we're updating the node version, I figure that could be considered a breaking change, hence 3.0.0